server/controllers/commentsController.js

```diff
@@ -0,0 +1,90 @@
+const Comment = require("../models/commentModel");
+const mongoose = require("mongoose");
+const asyncHandler = require("express-async-handler");
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider updating require style imports</b></div>
<div id="fix">

It looks like there are multiple `require()` style imports in this file. Consider using ES6 `import` statements instead for better TypeScript compatibility and consistency. Similar issues found at lines 1, 2, and 3 for importing `commentModel`, `mongoose`, and `express-async-handler`.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
import Comment from '../models/commentModel';
import mongoose from 'mongoose';
import asyncHandler from 'express-async-handler';
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Divyanshu9822/blog-mern-app/pull/22#issuecomment-2544587479>#245782</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged